test/test_checkpointer.jl

```diff
@@ -191,5 +203,23 @@ for arch in archs
         end
 
         run_checkpointer_cleanup_tests(arch)
+
+        # Run a simulation that saves data to a checkpoint
+        rm("initialization_test_iteration*.jld2", force=true)
+        simulation = initialization_test_simulation(arch, 4)
+        run!(simulation)
+
+        # Now try again, but picking up from the previous checkpoint
+        N = iteration(simulation)
+        checkpoint = "initialization_test_iteration$N.jld2"
+        simulation = initialization_test_simulation(arch, 8)
+        run!(simulation, pickup=checkpoint)
+
+        progress_cb = simulation.callbacks[:progress]
+        progress_cb.schedule.first_actuation_time
+        @test progress_cb.schedule.first_actuation_time == 4
+
+        rm("initialization_test_iteration*.jld2", force=true)
```

Comment:
fyi, no need to do that since these lines deal with that:

https://github.com/CliMA/Oceananigans.jl/blob/abb66e32d333562dd9aaeb7dd2ed8fac5e781368/docs/make.jl#L185-L202

Reply:
oh good we can delete then